clio-server/src/main/scala/org/broadinstitute/clio/server/ClioServer.scala

```diff
@@ -42,7 +42,11 @@ object ClioServer
     auditRequest & auditResult & completeWithInternalErrorJson & auditException & mapRejectionsToJson
   }
   private val innerRoutes: Route =
-    concat(authorizationRoute, statusRoutes, readGroupRoutes)
+    concat(
+      pathPrefix("api") { authorizationRoute },
+      statusRoutes,
+      pathPrefix("api") { readGroupRoutes }
```

Comment:
If we don't care about the order the top-level routes are concat-ed in, can we move `statusRoutes` up and just have:
```scala
pathPrefix("api") {
  authorizationRoute ~ readGroupRoutes
}
```